config/v1/types_infrastructure.go

```diff
@@ -169,6 +169,9 @@ const (
 
 	// AlibabaCloudPlatformType represents Alibaba Cloud infrastructure.
 	AlibabaCloudPlatformType PlatformType = "AlibabaCloud"
+
+	// NutanixPlatformType represents Nutanix infrastructure.
+	NutanixPlatformType PlatformType = "Nutanix"
```

Comment:
This value needs to be added to the `+kubebuilder:validation:Enum` annotation on the `PlatformType` type.

Reply:
@staebler  Added the value as requested.